test/unit/Query/QueryCollection.test.php

```diff
@@ -4,24 +4,71 @@
 use Gt\Database\Connection\DefaultSettings;
 use Gt\Database\Connection\Driver;
 use Gt\Database\Result\ResultSet;
+use PHPUnit_Framework_MockObject_MockObject;
 
 class QueryCollectionTest extends \PHPUnit_Framework_TestCase {
+/** @var  QueryCollection */
+private $SUT;
```

Comment:
Subject Under Test.  (I think) it's fairly standard notation in unit testing and calls-out the fact that this is the object that's being tested, rather than the fixtures around it.  There are scenarios where you'd have multiple instances of the same object in a test class, but you're only testing one of them.

Can change it if you'd rather though?